scheduler/clients/javascript/tests/requirements/requirements.spec.ts

```diff
@@ -380,12 +380,230 @@ describe('Requirements', () => {
       it.todo('To be implemented')
     })
 
-    describe("Users' calendars can be queried in groups", () => {
-      it.todo('To be implemented')
+    describe('Multiple calendars of the same user can be queried at once', () => {
```

Comment:
Not linked to the new feature of querying on multiple users for freebusy, but worth adding